content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: '.getDisplayName()' 
+Description: 'Returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.' 
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+ 
```

Comment:
```suggestion
  - 'Computer Science'
```

Reply:
I am not understanding the change here? Sorry, I am a newbie! Is it the spacing? And hey man I just realized I can't spell to save my life! I fixed the word 'calendar' ... as I had all the words spelled... 'calender' HAHHAHA, I seen you wrote it correctly as well on the Syntax header :D ty

Reply:
Yes that's correct I just removed the extra line after computer science.